### ISSUE TITLE
Request: Set charge rate 

### ISSUE THREAD

**Issue by remimarents** [Kemmey/Kemmey-TeslaWatch-Public#251]
I have 2 cars with 1 TWC and 1 generic wall connector, on a shared 32a circuit. So I need to dial down the charge current when the other car is charging, or else the breaker shuts off. Presently I manually change the rate on my car’s main screen- but it would be nice to do this from my watch, just like setting temperature and setting charge limit.
And- merry Christmas!

**Comment by Kemmey** [comment]
@remimarents I would love to add that feature, as I had need for it myself before I had proper electrical wiring installed. 

However, it's not possible to adjust with the api currently, unfortunately. 

My suggestion would be to manually program each car to start and stop charging at different hours during the night. That is also something I can look into adding to the app, but there are currently better options to do that. Charged App comes to mind :-D

**Comment by remimarents** [comment]
Yes, I can use schedule on my Tesla-  but I have some difficulty locating the screen on the other car: 
![4D06B237-BBC3-4768-A2DF-30379ACFFA39](https://user-images.githubusercontent.com/69077915/103087589-3e9ae580-45e8-11eb-8811-854967b97c51.jpeg)

**Comment by Kemmey** [comment]
Fair enough :-D
